tmpl/.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: status:undecided, type:task
+assignees: BurningEnlightenment
```

Comment:
dito

Reply:
Is just copy pasted from concrete. Did not look into the templates themself. Will update in a moment.